src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/PluginImpl.java

```diff
@@ -69,7 +79,8 @@
  * @author Robert Sandell &lt;robert.sandell@sonyericsson.com&gt;
  */
 @ExportedBean
-public class PluginImpl extends Plugin {
+@Extension
+public class PluginImpl extends GlobalConfiguration {
```

Comment:
Might need to return some form of special category so that it isn't included in the normal global configuration? But I guess it should work since it used to be a `Plugin` that should also be listed normally.

Reply:
> But I guess it should work since it used to be a Plugin that should also be listed normally.

Exactly. The empty jelly page is included.